frontend/packages/core/src/stories/buttongroup.stories.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import { action } from "@storybook/addon-actions"; // eslint-disable-line import/no-extraneous-dependencies
+import type { Meta } from "@storybook/react";
+
+import type { ButtonGroupProps } from "../button";
+import { ButtonGroup } from "../button";
+
+export default {
+  title: "Core/Button Group",
+  component: ButtonGroup,
+} as Meta;
+
+const Template = (props: ButtonGroupProps) => <ButtonGroup {...props} />;
+
+const sharedArgs = [{ text: "Back", onClick: action("onClick event") }, action("onClick event")];
```

Comment:
RE https://github.com/lyft/clutch/pull/607#discussion_r510354273, is this what you had in mind? I researched if there was a Storybook way to do this - tried setting the common entries at the component level (https://storybook.js.org/docs/react/writing-stories/args#component-args) but couldn't figure out how to merge both the component level args and the story-specific args.

Reply:
This works. I was thinking about just reusing the back button element and adding in the unique story buttons but having the action is ok as well.